src/box/lua/config/applier/box_cfg.lua

```diff
@@ -563,7 +566,6 @@ local function names_rw_watcher()
     return nil
 end
 
-
```

Comment:
This should be squashed into one of the previous commits

Reply:
Fixed